vnext/ReactCommon/TEMP_UntilReactCommonUpdate/jsi/jsi/test/testlib.cpp

```diff
@@ -1641,6 +1641,65 @@ TEST_P(JSITest, GetStringDataTest) {
   str.getStringData(rd, cb);
   EXPECT_EQ(buf, str.utf16(rd));
 }
+
```

Comment:
Q: Is this test commented out because JSI added a function? If so, we should still create an issue to track the implementation!